org.civicrm.bootstrapcivicrm/scss/bootstrap/mixins/_fa-icon.scss

```diff
@@ -1,17 +1,18 @@
-@mixin fa-icon($font-size: null, $content: null, $color: null) {
+@mixin fa-icon($args...) {
```

Comment:
Yes, this is a code improvement. The more I used this mixin, the more I felt it was not flexible enough, so I've refactored it. And yes, this way you're not forced to follow a particular order for the parameter, and just have to provide the ones you need.